pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -811,7 +813,18 @@ def solve(self, *args, **kwds):
             command.append("lf=" + str(logfile))
 
         try:
-            rc, txt = pyutilib.subprocess.run(command, tee=tee)
+            with subprocess.Popen(command, stdout=subprocess.PIPE,
+                                  stderr=subprocess.STDOUT, bufsize=1,
+                                  universal_newlines=True) as p, \
+                open('logfile', 'w') as f:
+                for line in p.stdout:
+                    if tee:
+                        sys.stdout.write(line)
+                    f.write(line)
```

Comment:
This is a problem, as the stdout does not appear until after the subprocess terminates (and is not the intent of `tee=True`).